distribution/integ-test-zip/build.gradle

```diff
@@ -35,5 +37,14 @@ publishing {
   }
 }
 
+Task integTestNetty4 = tasks.create("integTestNetty4", RestIntegTestTask) {
```

Comment:
I tried:

```diff
diff --git a/modules/transport-netty4/build.gradle b/modules/transport-netty4/build.gradle
index 613fa6c..78f9f24 100644
--- a/modules/transport-netty4/build.gradle
+++ b/modules/transport-netty4/build.gradle
@@ -1,9 +1,3 @@
-import com.sun.org.apache.bcel.internal.util.ClassPath
-
-import java.util.logging.LogManager
-import java.util.zip.Deflater
-import java.util.zip.Inflater
-
 /*
  * Licensed to Elasticsearch under one or more contributor
  * license agreements. See the NOTICE file distributed with
@@ -48,6 +42,14 @@ dependencies {
     compile "io.netty:netty-transport:4.1.4.Final-SNAPSHOT"
 }
 
+integTest {
+    includePackaged = true
+    cluster {
+        setting 'http.type', 'netty4'
+        setting 'transport.type', 'netty4'
+    }
+}
+
 thirdPartyAudit.excludes = [
         // classes are missing
 
@@ -164,5 +166,4 @@ thirdPartyAudit.excludes = [
         'io.netty.util.internal.shaded.org.jctools.util.JvmInfo',
         'io.netty.util.internal.shaded.org.jctools.util.UnsafeAccess',
         'io.netty.util.internal.shaded.org.jctools.util.UnsafeRefArrayAccess',
-
 ]
```

but it does not seem to cause the REST tests to run. What am I missing? For now I will leave the way that I have.